### PR TITLE
mobile-navbar

### PR DESCRIPTION
**Pull request by farukbigez**
-Mobile Navbar added
-.PNG logo changed as .SVG

### CODE REVIEW

File: src/App.test.tsx [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from 'react'
 import { render } from '@testing-library/react'
 import App from './App'
 
-test('renders learn react link', () => {
-    const { getByText } = render(<App />)
-    const linkElement = getByText(/learn react/i)
-    expect(linkElement).toBeInTheDocument()
-})
+// test('renders learn react link', () => {
```
Comment (nopestack) [inline src/App.test.tsx:5]: Please add real unit tests here to verify that `App` is rendering correctly.

File: src/components/NavigationBar.tsx [hunk 2]
```diff
@@ -23,10 +74,19 @@ const NavigationBar = ({ items }: { items: NavigationItem[] }) => {
     const { isAuthenticated, loginWithRedirect, logout } = useAuth0()
 
     return (
-        <Flex pl={5} pr={5} color="black" alignItems="center" bg="#fff">
+        <Flex color="black" alignItems="center" bg="#fff">
```
Comment (nopestack) [inline src/components/NavigationBar.tsx:77]: The padding on this Navbar is a bit off and I believe `master` was updated with a fix for this, would you mind doing `git pull origin master` while inside your local branch?

**Review by nopestack — COMMENTED** [review]
Please sync w/ the design team about the intended mobile UI of the Navbar.

**Review by tdubreucq — COMMENTED** [review]
1) Is the menu linked to the Hamburger component part of this task? If it is, pressing the hamburger doesn't have any effect for now
2) Some phone resolutions are messing with the navbar (see screenshots) 
image
Also tablet resolutions are not handled, need to know if it's part of the task scope too or not

**Comment by farukbigez** [comment]
> 1. Is the menu linked to the Hamburger component part of this task? If it is, pressing the hamburger doesn't have any effect for now
> 2. Some phone resolutions are messing with the navbar (see screenshots)
>    image
>    Also tablet resolutions are not handled, need to know if it's part of the task scope too or not

I was not know responsible with Side Drawer. I'm gonna add this ASAP. I was dealing with tablet side responsiveness and I'm gonna create new req about that.

**Comment by nopestack** [comment]
I'm closing this as it has not been updated for more than a week

**Pull request closed without merging**